api/hasura/customTypes/createCircleCustomTypes.ts

```diff
@@ -0,0 +1,47 @@
+export type Maybe<T> = T | null;
+export type Exact<T extends { [key: string]: unknown }> = {
+  [K in keyof T]: T[K];
+};
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+};
+
+export type Create_Circle_Input = {
+  user_name: Scalars['String'];
+  address: Scalars['String'];
+  circle_name: Scalars['String'];
+  protocol_name?: Maybe<Scalars['String']>;
+  protocol_id?: Maybe<Scalars['Int']>;
+};
+
+export type Create_Circle_Response = {
```

Comment:
If you run `yarn generate`, I hope these get created automagically by zeus + hasura introspection.

We don't want to manually manage these types.

Reply:
it did create a`ValueTypes['create_circle_input']` type have removed this since its unneeded
But the fetch error is somehow back after i updated from main, can't query/mutate anymore from backend

Reply:
This is amazing